src/components/Board/_Board.scss

```diff
@@ -1,27 +1,54 @@
+.Board {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .turn-status {
+    margin-bottom: 12px;
+    margin-top: -15px;
+    font-size: 20px;
+    font-weight: 300;
+    font-family: 'Krona One', sans-serif;
+  }
+}
+
 .gameboard {
-  box-sizing: border-box;
+  background: rgb(21, 21, 24);
```

Comment:
Do we use these colors more than once?
- If so, then it might be a good idea for us to look into using our scss variables